exdb/models.py

```diff
@@ -77,6 +76,15 @@ def needs_evaluation(self):
         return self.status == 'ad' and self.end_datetime <= now()
 
 
+class ExperienceApproval(models.Model):
+    experience = models.ForeignKey(Experience, related_name='approval_queue')
+    approver = models.ForeignKey(settings.AUTH_USER_MODEL)
+    timestamp = models.DateTimeField(default=now)
```

Comment:
`timestamp` should have `blank=True` to make editing/creating `ExperienceApproval`s via ModelForm easier.